include/vcpkg/base/format.h

```diff
@@ -25,7 +25,7 @@ namespace fmt
     template<>
     struct formatter<vcpkg::LineInfo, char>
     {
-        constexpr auto parse(format_parse_context& ctx) const -> decltype(ctx.begin())
+        static constexpr auto parse(format_parse_context& ctx) -> decltype(ctx.begin())
```

Comment:
I don't believe this change is safe because we are following fmt's protocol and it is going to call with `a.parse(` syntax, not `A::parse` syntax.

Reply:
Seems to have no effect, but I will revery just in case.